src/avfoundation.cs

```diff
@@ -5540,7 +5540,7 @@ interface AVMetadata {
 		NSString FormatHlsMetadata { get; }
 #endif
 
-		[iOS (9,3)][Mac (10, 11, 3)]
+		[iOS (9,3)][Mac (10, 12)]
```

Comment:
what's the exact version of 10.11 on the bots ?
Our tests shows ` OS Version: Unix 15.6.0.0` so I'm guessing it's the latest El Capitan OS X v10.11.6
note: might be nice to add something like `uname -a` to the bot's logs

Reply:
On `xam-macios-capitan-1`:

```shell
$ uname -a
Darwin xam-macios-capitan-1 15.6.0 Darwin Kernel Version 15.6.0: Thu Jun 23 18:25:34 PDT 2016; root:xnu-3248.60.10~1/RELEASE_X86_64 x86_64
```